code/modules/antagonists/cult/runes.dm

```diff
@@ -229,8 +229,13 @@ structure_check() searches for nearby cultist structures required for the invoca
 		do_sacrifice(L, invokers)
 	animate(src, color = oldcolor, time = 5)
 	addtimer(CALLBACK(src, /atom/proc/update_atom_colour), 5)
+	Cult_team.check_size() // Triggers the eye glow or aura effects if the cult has grown large enough relative to the crew
 	rune_in_use = FALSE
-
+/proc/mouse_test(mob/user)
+	var/list/paramlist = params2list(user.client.mouseParams)
+	for(var/i in paramlist)
+		to_chat(world, "Param is [i] with contents [paramlist[i]]")
```

Comment:
you can just delete this, looks like debug hooplah

that or move it to... click *shudders*